tests/test_utils.py

```diff
@@ -138,4 +266,31 @@ def test_find_moodle_media_references(mbz_path):
     for act in activities:
         media_references += utils.find_moodle_media_references(act)
 
-    assert set([MOODLE_VIDEO_FILE, MOODLE_TRACK_FILE]) == set(media_references)
+    assert set([MOODLE_VIDEO_FILE, MOODLE_TRACK_FILE]) == \
+        set(media_references)
+
+
+def test_find_question_html(mbz_path):
```

Comment:
Just as another example, you can remove `squish_list()` here too I think and this can be:

```python
def test_find_question_html(mbz_path):
    html = utils.parse_question_bank_for_html(mbz_path)
    assert len(html) == 4

    expected = [
        QUESTION1_CONTENT,
        QUESTION2_CONTENT,
        ANSWER1_CONTENT,
        ANSWER2_CONTENT
    ]
    assert set(html) == set(expected)
```